openassessment/xblock/test/test_staff.py

```diff
@@ -274,6 +275,28 @@ def test_assessment_error(self, xblock):
 class TestStaffTeamAssessment(StaffAssessmentTestBase):
     """ Test Staff Team Assessment Workflow"""
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.expected_answer = {
+            'points_earned': 2,
+            'parts0_criterion_name': 'Form',
+            'parts1_criterion_name': 'Concise',
+            'parts2_criterion_name': 'Clear-headed',
+            'parts0_option_name': 'Facebook',
+            'parts1_option_name': 'HP Lovecraft',
+            'parts2_option_name': 'Yogi Berra'
+        }
+        cls.regrade_expected_answer = {
+            'points_earned': 3,
```

Comment:
where are these points_earned coming from?

Reply:
from tallying up the answer. Essentially I am following the same pattern as other tests in the class.